server/src/main/java/app/cash/quickjs/QuickJs.java

```diff
@@ -0,0 +1,39 @@
+package app.cash.quickjs;
+
+import javax.script.ScriptEngine;
+import javax.script.ScriptEngineManager;
+import java.io.Closeable;
+
+public final class QuickJs implements Closeable {
+    private final ScriptEngine engine;
+
+    public static QuickJs create() {
+        return new QuickJs(new ScriptEngineManager());
+    }
+
+    public QuickJs(ScriptEngineManager manager) {
+        this.engine = manager.getEngineByName("js");
+    }
+
+    public Object evaluate(String script, String fileName) {
+        try {
+            return engine.eval(script);
+        } catch (Exception exception) {
+            exception.printStackTrace();
+            return "";
+        }
+    }
+
+    public Object evaluate(String script) {
+        try {
+            return engine.eval(script);
+        } catch (Exception exception) {
```

Comment:
Is there a reason to swallow the error? I think that the sources would just error anyway if it returns a empty string since its not the expected output.

Reply:
I test it with manhuagui extension and it work well. But i find Readcomiconline extension will use quickjs's compile function and i have no idea about how this function works.
![image](https://user-images.githubusercontent.com/24984102/194811795-7024251b-4749-472c-919c-af9adc3f600c.png)

And you are right, I should not swallow the error, may need to throw another runtime exception. I just want the function signature to be same with quickjs.